shoppingcard.css

```diff
@@ -0,0 +1,142 @@
+* {
+
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+
+}
```

Comment:
don't write common css in every file. write it in a separate file then import it here.